lib/iruby/command.rb

```diff
@@ -1,19 +1,37 @@
 require 'shellwords'
-require 'fileutils'
+require 'pathname'
 
 module IRuby
   class Command
     WINDOWS_REGEXP = /mswin(?!ce)|mingw|cygwin/
-    IRUBYDIR = RUBY_PLATFORM =~ WINDOWS_REGEXP ? '~/iruby' : '~/.config/iruby'
 
     def initialize(args)
       @args = args
+
+      @ipython_dir = ENV['IPYTHONDIR'] || '~/.ipython'
+      @args.each do |arg|
+        @ipython_dir = $1 if arg.match(/\A--ipython-dir=(.*)\Z/)
+      end
+      @ipython_dir = Pathname.new(@ipython_dir).expand_path
     end
 
     def run
-      raise 'Use --iruby-dir instead of --ipython-dir!' unless @args.grep(/\A--ipython-dir=.*\Z/).empty?
-
-      if @args.first == 'kernel'
+      case @args.first
+      when 'version', '-v', '--version'
+        require 'iruby/version'
+        puts IRuby::VERSION
+      when 'help', '-h', '--help'
+        print_help
+      when 'register'
+        if ipython_register_file.exist? and not @args.include? "--force"
```

Comment:
Something stylistic: I would prefer if you use single quotes if the string doesn't include interpolations.